node-api-dotnet/samples/NodeKernelSyntaxExamples/README.md

```diff
@@ -9,6 +9,23 @@ This project enables advanced interoperability between .NET and JavaScript in th
 To run the samples, first set the following environment variables referencing your
```

Comment:
A pre-requisite section after the intro would be helpful with the minimum required version of NodeJS.